test/GitHub.InlineReviews.UnitTests/Models/DiffUtilitiesTests.cs

```diff
@@ -269,5 +315,33 @@ public void NoLineMatchesFromNoLines()
                 Assert.Equal(null, line);
             }
         }
+
+        public class TheLineReaderClass
+        {
+            [Theory]
+            [InlineData("", "", null, null)]
```

Comment:
Why not pass `new string[] { "", null }` directly into the test?

Reply:
I original wrote it using using NUnit, which allows test methods to accept `params string[] lines`. When I converted it to xUnit, I didn't realize you could define an array inside an attribute.